.travis.yml

```diff
@@ -6,7 +6,7 @@ matrix:
   include:
     - if: repo = arvidn/libtorrent
       env: variant=release sonar_scan=1 toolset=gcc
-    - env: variant=test_debug lint=1 tests=1 toolset=gcc-sanitizer
+    - env: variant=test_debug lint=1 pylint=1 tests=1 toolset=gcc-sanitizer
```

Comment:
I would prefer to do this in the same job as `sonar_scan=1`, as it only takes 2-3 minutes right now